classes/04class/exercises/c04-iac01/chisholm-ray/iam.tf

```diff
@@ -0,0 +1,69 @@
+data "aws_iam_policy_document" "policy_document" {
+  statement {
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:ListAllMyBuckets"
+    ]
+
+    resources = [
+        "*"
```

Comment:
Suggestion : Please check if this supports least privilege IAM policy practices.
[Ref 1](https://aws.amazon.com/blogs/security/techniques-for-writing-least-privilege-iam-policies/)
[Ref 2](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
[Ref 3](https://aws.amazon.com/premiumsupport/knowledge-center/s3-console-access-certain-bucket/)

By using the wildcard `*`, does EC2 get the ability to list other buckets (assume there are many S3 buckets) as well ?